src/lib.rs

```diff
@@ -130,33 +130,27 @@ impl Metadata {
 /// Capability to convert a key into an node index.
 pub trait Key {
     /// Convert the key `self` to a `usize` index.
-    fn find<M: Miniconf>(&self) -> core::option::Option<usize>;
+    fn find<const D: usize, M: Miniconf<D>>(&self) -> core::option::Option<usize>;
```

Comment:
Can this also have a default value for the const-generic? Not sure if rust supports that in the position.

Reply:
We'd have to pull D up into the trait. It doesn't help and would be longer. We always have to explicitly carry the depth anyway.